### PR TITLE
issue #14: documenting how to run nachet-backend

### LINKED ISSUE #14: Issue #14: documenting how to run nachet-backend
Documenting the procedure of running nachet-backend either as a devcontainer or from the Dockerfile, and running tests.

### PR DESCRIPTION
**Pull request by RussellJimmies**
Amending the documentation to detail how to run the project from a devcontainer or from the dockerfile, and how to run the tests.

Closes #14

### CODE REVIEW

**Comment by rngadam** [comment]
your branch description should use `issue #14` to autolink to the issue, not the branch name (which we already know from the PR description)

File: README.md [hunk 1]
```diff
@@ -3,3 +3,28 @@
...
+
+### RUNNING NACHET-BACKEND AS A DOCKER CONTAINER
+If you want to run the program as a Docker container (e.g., for production), use: 
+```
+docker build -t nachet-backend .
+docker run -p 8080:8080 -v $(pwd):/app nachet-backend
+```
+
+### TESTING NACHET-BACKEND
+To test the program, simply run the `run_tests.py` file.
```
Comment (rngadam) [inline README.md]: The canonical way to run the tests should be the same as in Github workflow:

https://github.com/ai-cfia/github-workflows/blob/3af315f654084b997254c75072dce58ecaa2c629/.github/workflows/workflow-lint-test-python.yml#L36

`python -m unittest discover -s tests`

File: README.md [hunk 2]
```diff
@@ -3,3 +3,28 @@
...
+- All HTTP requests are handled in `app.py` in the root folder
+- Azure Storage API calls are handled in the `azure_storage_api/azure_Storage_api.py
+- Inference results from model endpoint are directly handled in `model_inference/inference.py`
+
+****
+ 
+### RUNNING NACHET-BACKEND FROM DEVCONTAINER
+When you are developping, you can run the program while in the devcontainer by using this command:
+```
+hypercorn -b :8080 app:app
```
Comment (rngadam) [inline README.md:19]: let's start assigning default ports across our backends/frontend that won't conflict with each other in a PORTS.md document in dev-rel-docs: https://github.com/ai-cfia/dev-rel-docs

**Review by rngadam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RussellJimmies) [reply README.md]: Roger that!

File: README.md (hunk 2 above)
Comment (RussellJimmies) [reply README.md:19]: I will create a separate Issue for this

**Review by rngadam — APPROVED** [review]

**Pull request merged**